src/tabular-api-surface.ts

```diff
@@ -532,6 +567,10 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
               numConsecutiveErrors,
               backOffSettings
             );
+            metricsCollector?.onAttemptComplete(
```

Comment:
I see a `return` on line 553. Don't you need to finalize things there?

Reply:
Good catch. Added that fix!